website/docs/intro.md

```diff
@@ -2,7 +2,18 @@
 sidebar_position: 1
 ---
 
-# Introduction
-
-> Work in progress...
+# General
```

Comment:
Maybe rename "General" to "About Plugin"? 
But if we'll write on this page not only about plugin but also about "how to use these docs", isn't it better to leave "Introduction" here?

Reply:
Yes the introduction sounds better.